src/command.js

```diff
@@ -30,9 +29,7 @@ export default class Command <Flags: InputFlags> {
   static _version: pjson.version
 
   static get id (): string {
-    let cmd = this.command ? `${this.topic}:${this.command}` : this.topic
-    if (this.namespace) cmd = `${this.namespace}:${cmd}`
-    return cmd
+    return this.topic ? `${this.topic}:${this.command}` : this.command
```

Comment:
even though flow enforces this, can you still add some logic in to make it work if command is not defined but topic is? otherwise I'm pretty sure this will break things